go/teams/service_helper.go

```diff
@@ -314,8 +314,18 @@ func RemoveMember(ctx context.Context, g *libkb.GlobalContext, teamname, usernam
 	if err != nil {
 		return err
 	}
+
+	// if username looks like an email address, then remove an invite
+	// (the loadUserVersionByUsername will fail with an email address)
+	if libkb.CheckEmail.F(username) {
```

Comment:
what happens if it's `foo@github`?

Reply:
So do you distinguish between `excite.com` and `twitter` on the basis of the existence of the `.com`?  Just feels a little brittle, but I guess we can get away with it.

Reply:
Will work as long as social types don't contain `.`.

Reply:
ok, i think that's ok. it feels a bit fragile

Reply:
The alternative is to change `remove-member` to specify an email address.  `keybase team remove-member --email=blah@blah.com`

Reply:
@malgorithms?  IT would be symmetric with the invite.  I'm up for whatever is the best UX and I won't worry about the fragility for now

Reply:
I think I like the symmetry of `--email`. I just wrote this to patrick in keybase chat, but the important thing around all this is getting actually helpful error messages. Exmaples:

```
> keybase team remove-member --email=jack@jack.com
Error: there is currently no pending invitation for jack@jack.com. If that person is already on your team, please remove them with their keybase username. 
```

```
> keybase team remove-member --user=jack@jack.com
Error! If you'd like to cancel a pending invite by email address, please use `--email` instead of `--user`. If you'd like to remove an existing member from your team, please use their keybase username.
```

Reply:
If we need errors that look like that, then we should probably plumb the error back up to the client and format it there. I'm not a fan of moving knowledge of what the CLI flags are into the `teams` dir, since it's a lot easier to change the flags and not the errors; or that the errors won't make sense if plumbed through to electron

Reply:
totally agree